modules/oe_paragraphs_chart/config/install/core.entity_form_display.paragraph.oe_chart.default.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.oe_chart.field_oe_media
+    - paragraphs.paragraphs_type.oe_chart
+id: paragraph.oe_chart.default
+targetEntityType: paragraph
+bundle: oe_chart
+mode: default
+content:
+  field_oe_media:
+    weight: 0
+    settings:
+      match_operator: CONTAINS
+      match_limit: 10
```

Comment:
This value was added in Drupal 8.8, which means that if someone enables this module on 8.7 the schema validation will fail (or should at least). Even if this is an optional module, given the short time to live of 8.7 (8.9 is weeks away), let's require on the main project Drupal core 8.8.

Reply:
Done and also applied to other submodules.